sunflower-land/sunflower-land
# [FEAT] Faction Point Cutover

# Description

Disables Faction Points when Kingdom goes live ( 14th June). Introduces new ERC 1155 resource `Faction Mark`. Different features behave differently:

**Chores**
Faction points are removed from chores

**Deliveries**
Faction points are removed from deliveries

**Chicken Rescue**
Starts putting out Marks

**Donations**
Feature is disabled

## How to Test

All features are toggled by a single switch `FACTION_POINT_CUTOFF`. In order to test use `FACTION_POINT_CUTOFF` in the following scenarios:

1. `FACTION_POINT_CUTOFF` in future.

2. `FACTION_POINT_CUTOFF` in past.

Visit each of the four features listed above to make sure they indeed do not give out faction points.

## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?

localhost

# Checklist:
